### PR TITLE
#180 fix iso 25010 hyperlinks

### PR DESCRIPTION
**Pull request by gernotstarke**
maaany references to iso25010.com removed.

(

### CODE REVIEW

**Review by alxlo — APPROVED** [review]
Great :-)

File: docs/1-terms/I/term-iso-25010.adoc [hunk 1]
```diff
@@ -3,21 +3,35 @@
...
 ==== ISO 25010
 
-Standards to describe (and evaluate) _software product quality_.
-"The quality model determines which quality characteristics will be taken into account when evaluating the properties of a software product." (quote from the link:https://iso25000.com/index.php/en/iso-25000-standards/iso-25010[ISO website])
+Official name:
 
-For a list of quality attributes defined by the ISO 25010 standard, refer to <<ref-iso-25010>>
+Systems and software engineering — Systems and software Quality Requirements and Evaluation (SQuaRE)
+
+It is a standards to describe _software product quality_.
```
Comment (programming-wolf) [inline docs/1-terms/I/term-iso-25010.adoc:10]: *standard (singular)

File: docs/1-terms/I/term-iso-25010.adoc [hunk 2]
```diff
@@ -3,21 +3,35 @@
...
 
 // tag::DE[]
 ==== ISO 25010
 
-Normen zur Beschreibung (und Bewertung) von *Softwareproduktqualität*.
-Das Qualitätsmodell legt fest, welche Qualitätseigenschaften bei der
-Bewertung der Eigenschaften eines Softwareprodukts berücksichtigt
-werden. (Übersetztes englisches Zitat von der
-link:https://iso25000.com/index.php/en/iso-25000-standards/iso-25010[ISO-Website])
+N
```
Comment (programming-wolf) [inline docs/1-terms/I/term-iso-25010.adoc:25]: N weg

File: docs/1-terms/I/term-iso-25010.adoc [hunk 3]
```diff
@@ -3,21 +3,35 @@
...
+Systems and Software Engineering - Systems and Software Quality Requirements and Evaluation (SQuaRE)
+
+Es handelt sich um eine Norm zur Beschreibung der _Softwareproduktqualität_.
+
+ISO schlägt ein hierarchisches Modell der Produktqualität vor, mit derzeit (Standardversion 2011) 8 Top-Level-Attributen.
+
+image::1_architecture/ISO-25010-EN.png
```
Comment (programming-wolf) [inline docs/1-terms/I/term-iso-25010.adoc:32]: Sollte hier das deutsche Bild verlinkt werden?

**Review by programming-wolf — COMMENTED** [review]
Some remarks.

**Comment by programming-wolf** [comment]
I had some remarks of mistakes that were made. Hopefully they're not lost now...

**Pull request merged**